src/commands/android/subcommands/common.ts

```diff
@@ -0,0 +1,49 @@
+import colors from 'ansi-colors';
+
+import Logger from '../../../logger';
+import {AVAILABLE_SUBCOMMANDS} from '../constants';
+
+function showHelp(subcommand: string) {
+  const subcmd = AVAILABLE_SUBCOMMANDS[subcommand];
+
+  Logger.log(`Usage: ${colors.cyan(`npx @nightwatch/mobile-helper android ${subcommand} [options]`)}`);
+  Logger.log();
+  Logger.log(colors.yellow('Options:'));
+
+  const longest = (xs: string[]) => Math.max.apply(null, xs.map(x => x.length));
+
+  if (subcmd.options && subcmd.options.length > 0) {
+    const optionLongest = longest(subcmd.options.map(option => `--${option.name}`));
+    subcmd.options.forEach(option => {
+      const optionStr = `--${option.name}`;
+      const optionPadding = new Array(Math.max(optionLongest - optionStr.length + 3, 0)).join('.');
+      Logger.log(`    ${optionStr} ${colors.grey(optionPadding)} ${colors.gray(option.description)}`);
+    });
```

Comment:
This logic is repeated at a log of places. We should probably create a utils function out of it and call it from a common module.